website/pages/community/index.jsx

```diff
@@ -19,27 +19,27 @@ export default function CommunityPage() {
           {
             header: 'Community Forum',
             body:
-            '[Vagrant Community Forum](https://discuss.hashicorp.com/c/vagrant/24)',
+              '<a href="https://discuss.hashicorp.com/c/vagrant/24">Vagrant Community Forum</a>',
```

Comment:
`vertical-text-block-list` doesn't contain markdown processing anymore, so we need to pass it HTML. This could be done automatically in `getStaticProps` if we feel it'd be worth it for authoring convenience.